classes/site/Version.inc.php

```diff
@@ -284,6 +285,47 @@ function getVersionString($numeric = true) {
 
 		return $numericVersion;
 	}
+
+	/**
+	 * Checks if this version is compatible with given version
+	 * Returns:
+	 * 		true iff the version given is compatible with this version
+	 * 		false iff the version given is incompatible with this version
+	 * @param $version string/Version the version to compare against
+	 * @return boolean
+	 */
+	function isCompatible($version) {
+		$versionParser = new VersionParser();
+		$semver = new Sem();
+		$parsed = $versionParser->parseConstraints($version);
+		$systemVersion = $this->getVersionString();
+		if ($parsed instanceof Composer\Semver\Constraint\MultiConstraint) {
```

Comment:
Why do multiple constraints need to be examined separately?   Will a direct `return $semver->satisfies($systemVersion, $parsed);` not suffice?